SpatialGDK/Source/SpatialGDKServices/Private/SSpatialOutputLog.cpp

```diff
@@ -313,38 +292,33 @@ void SSpatialOutputLog::FormatAndPrintRawLogLine(const FString& LogLine)
 		FormatAndPrintRawErrorLine(LogLine);
 		return;
 	}
+	FString LogCategory = LogMatcher.GetCaptureGroup(1);
+	FString LogLevelText = LogMatcher.GetCaptureGroup(2);
+	FString LogMessage = LogMatcher.GetCaptureGroup(3);
 
-	FString LogLevelText = LogMatcher.GetCaptureGroup(1);
-	FString LogMessage = LogMatcher.GetCaptureGroup(2);
-	FString LogCategory = LogMatcher.GetCaptureGroup(4);
+	const FRegexPattern WorkerLogPattern = FRegexPattern(TEXT("\\[(.*)\\] (.*)"));
+	FRegexMatcher WorkerLogMatcher(WorkerLogPattern, LogMessage);
 
-	// For worker logs 'WorkerLogMessageHandler' we use the worker name as the category. The worker name can be found in the msg.
-	// msg=[WORKER_NAME:WORKER_TYPE] ... e.g. msg=[UnrealWorkerF5C56488482FEDC37B10E382770067E3:UnrealWorker]
-	if (LogCategory == TEXT("WorkerLogMessageHandler") || LogCategory == TEXT("Runtime"))
+	if (WorkerLogMatcher.FindNext())
 	{
-		const FRegexPattern WorkerLogPattern = FRegexPattern(TEXT("\\[([^:]*):([^\\]]*)\\] (.*)"));
-		FRegexMatcher WorkerLogMatcher(WorkerLogPattern, LogMessage);
-
-		if (WorkerLogMatcher.FindNext())
-		{
-			LogCategory = WorkerLogMatcher.GetCaptureGroup(1);		  // Worker Name
-			FString WorkerType = WorkerLogMatcher.GetCaptureGroup(2); // Worker Type
-
-			if (LogCategory.StartsWith(WorkerType))
-			{
-				// We shorten the category name to make it more human readable. e.g. UnrealWorkerF5C56
-				LogCategory = LogCategory.Left(WorkerType.Len() + 5);
-			}
-
-			LogMessage = WorkerLogMatcher.GetCaptureGroup(3);
-		}
+		FString LogMessageCategory = WorkerLogMatcher.GetCaptureGroup(1);
+		LogMessage = WorkerLogMatcher.GetCaptureGroup(2);
+		LogCategory = LogMessageCategory.Left(20);
+	}
+	else
+	{
+		// If the Log Category is not of type Worker, then it should be categorised as Runtime instead
+		LogCategory = TEXT("Runtime");
```

Comment:
Let's make this into a const for this class, you'll probably use it above in the `FormatAndPrintRawErrorLine`